indexing-service/src/test/java/org/apache/druid/indexing/common/task/TaskReportSerdeTest.java

```diff
@@ -55,7 +55,8 @@ public void testSerde() throws Exception
             ImmutableMap.of(
                 "number", 1234
             ),
-            "an error message"
+            "an error message",
+            false
```

Comment:
Testing with `true` would be better because missing booleans are defaulted to false by Jackson in Druid.